include/dtpcontrols/FlowMasterNode.hpp

```diff
@@ -31,6 +31,16 @@ class FlowMasterNode : public uhal::Node {
   explicit FlowMasterNode(const uhal::Node& );
   virtual ~FlowMasterNode();
 
+  void source_select(const std::string&, const bool);
+  void source_select(const uint32_t&, const bool);  
+  void sink_select(const std::string& , const bool);
+  void sink_select(const uint32_t&, bool);
+
+private:
+
+  bool m_exists(const uint32_t&, const std::vector<uint32_t>&);
```

Comment:
these methods do not rely on the object state, so toolbox.cpp would be a more appropriate place for them.